lib/repo-type-mappings/yocto-based-OS-image/versionist.conf.js

```diff
@@ -49,7 +49,7 @@ module.exports = {
     return 'patch'
   },
   incrementVersion: (currentVersion, incrementLevel) => {
-    const revision = Number(currentVersion[currentVersion.length - 1])
+    const revision = Number(currentVersion.split(/(rev|\.)/).pop());
```

Comment:
If this is a `rev+1`, revision will be 1, if it's `0.0.2` revision will be 2. Is this the behaviour we want? Or is the latter not possible here...

Reply:
This is the behaviour we want based on https://github.com/product-os/balena-versionist/pull/63